code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -714,3 +714,90 @@
 		segment_length = armsy.get_length()
 		qdel(armsy)
 		return
+
+/obj/effect/proc_holder/spell/pointed/void_blink
+	name = "Void Phase"
+	desc = "Let's you blink to your pointed destination, causes 3x3 aoe damage bubble around your pointed destination and your current location"
+	invocation_type = INVOCATION_WHISPER
+	invocation = "RE'L'TY PH'S'E"
+	clothes_req = FALSE
+	range = 9
+	action_background_icon_state = "bg_ecult"
+	charge_max = 300
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "voidblink"
+	selection_type = "range"
+
+/obj/effect/proc_holder/spell/pointed/void_blink/can_target(atom/target, mob/user, silent)
+	. = ..()
+	if(get_dist(get_turf(user),get_turf(target)) < 2 )
+		return FALSE
+
+/obj/effect/proc_holder/spell/pointed/void_blink/cast(list/targets, mob/user)
+	. = ..()
+	var/target = targets[1]
+	var/turf/targeted_turf = get_turf(target)
+
+	playsound(user,'sound/magic/voidblink.ogg',100)
+	playsound(targeted_turf,'sound/magic/voidblink.ogg',100)
+
+	new /obj/effect/temp_visual/voidin(user.drop_location())
+	new /obj/effect/temp_visual/voidout(targeted_turf)
+
+	for(var/mob/living/living_mob in range(1,user)-user)
+		if(IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
+			continue
+		living_mob.adjustBruteLoss(40)
+
+	for(var/mob/living/living_mob in range(1,targeted_turf)-user)
+		if(IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
+			continue
+		living_mob.adjustBruteLoss(40)
+
+	user.forceMove(targeted_turf)
```

Comment:
you should probably use do_teleport() instead of forcemove, or else this'll be able to teleport you in places where you shouldn't be teleporting (like centcom).